src/content/configuration/node.md

```diff
@@ -46,38 +35,40 @@ module.exports = {
 };
 ```
 
-Since webpack 3.0.0, the `node` option may be set to `false` to completely turn off the `NodeStuffPlugin` and `NodeSourcePlugin` plugins.
+Since webpack 3.0.0, the `node` option may be set to `false` to completely turn off the `NodeStuffPlugin` plugin.
 
 ## `node.global`
 
-`boolean = true`
+`boolean`
 
-Defaults to `false` for [targets](/configuration/target/) `node`, `async-node` and `electron-main`.
```

Comment:
Why remove this? Is there a summary of [`TARGETS`](https://github.com/webpack/webpack/blob/32bd1a2ff53c6b7fbf99d0e473d6db98f9e3a3f2/lib/config/target.js#L75-L269) I'm missing, or...?

Reply:
I don't remember the specific reason. There're more than three targets having `node.global` set to `false`, which could be the reason I removed this.

Reply:
The deleted info is outdated, indeed, but why not update it, instead of deleting it? How can I find out what it defaults to without digging into the code?

I'm concerned about this because I actually had an old todo, about improving this page, and by the time I got around to it, somebody already added the defaults, and when I went to look who did it, out of curiosity, I find that it got deleted :smile:

Reply:
pr welcome.

Reply:
This doesn't encourage me to contribute in this particular case. As it stands, it'll just get deleted a few months later again because of being slightly inaccurate.

Reply:
Yeah, it's difficult to align with all changes happening in webpack core project. There will always some stale contents.